### DISCUSSION TITLE
How to group nvm, Dockerfile and devcontainers udpates ?

### DISCUSSION THREAD

**Discussion by Djiit** [renovatebot/renovate#30530] [Request Help] [answered]
### What would you like help with?

None

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

38

### Please tell us more about your question or problem

Hello team,

I've got a simple use case where I need to update Node.JS version in both .nvmrc, Dockerfile and devcontainer.json at the same time to avoid issues. Is there any packageRules setting for this ?

Thank you.

**Comment by RahulGautamSingh** [comment, marked as answer]
Use any suitable matchers in [packageRules ](https://docs.renovatebot.com/configuration-options/#packagerules) available along with [groupName](https://docs.renovatebot.com/configuration-options/#groupname)

**Reply by RahulGautamSingh** [reply]
most probably `matchPackageNames` will work for you

**Reply by Djiit** [reply]
I'll try this and report back. Still unsure how to make sure the versions bumps are correct (e.g. all the PR bumps the same Node.js version)

**Reply by RahulGautamSingh** [reply]
Won't the latest version for the same package ie. `node` be the same across all the registires ie. npm, docker-hub.

Lets say a pr was created with all 3 grouped then you just need to wait a bit till they sync as we rebase the pr incase new updates are available.

**Reply by Djiit** [reply]
Thx. Wil investigate and reopen if needed.
